engine/src/test/java/io/camunda/zeebe/engine/processing/deployment/CreateDeploymentTest.java

```diff
@@ -224,6 +226,39 @@ public void shouldCreateDeploymentResourceWithMultipleProcesses() {
             Bpmn.convertToString(process).getBytes(), Bpmn.convertToString(process2).getBytes());
   }
 
+  @Test
+  public void shouldDoAtomicDeployments() {
+    // given
+    final String process1Id = "process1";
+    final BpmnModelInstance process1 =
+        Bpmn.createExecutableProcess(process1Id)
+            .startEvent()
+            .businessRuleTask("invalid_business_rule_task_id")
```

Comment:
This Business Rule Task is invalid only because it's missing the Called Decision. It's unclear to the reader what is invalid, unless they know this requirement.

❌ We should use something else to clarify that it is an invalid process. Let me know if you want any tips.

Reply:
Yes, your point definitely makes sense. Updated using process without start event. That should be more trivial to understand. I also rename process instance variables to make it explicit.